package.json

```diff
@@ -11,14 +11,21 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
+    "lint": "eslint --ext .js,.jsx ./",
+    "test": "CI=true react-scripts test --env=jsdom",
```

Comment:
by default, the `test` script runs in "watch" mode, which is good for development, but I read we need "CI mode" to just run the tests plain (like we'll need for pre-commit hooks, builds, etc).

basically this just gives a pass/fail.